fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dto/Parameter.java

```diff
@@ -220,6 +220,9 @@ public void visit(IParameterVisitor visitor) throws FHIRPersistenceException {
         case URI:
             visitor.stringValue(name, this.valueString, baseLevel);
             break;
+        case SPECIAL:
```

Comment:
i think there is one other parameter with type=special ...maybe we should verify that we have a 'near' search parameter here (just to be safe)

Reply:
I checked this out using the search-parameters.json and the website.

Reply:
only 1

Reply:
2 listed at https://www.hl7.org/fhir/search.html#special
* near
* _filter

but yeah, if `_filter` doesn't appear in search-parameters.json then i guess we're good

Reply:
it is not.